src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/InputJavadocPackageBadCls.java

```diff
@@ -1,5 +1,9 @@
-package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocpackage;
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocpackage;      // violation
 
+/* Config:
+ * allowLegacy=false
```

Comment:
Please use `"` as delimiter around property as you have done below for `fileExtensions="java"`. This goes for all following `allowLegacy` properties.

Reply:
Done.

Reply:
Resolved.